doc/sphinx/source/project/introduction.rst

```diff
@@ -4,189 +4,97 @@
 Introduction
 ############
 
+For making further developments in Artificial Intelligence (AI), developers need better methods to accelerate the performance of deep learning. Deep learning models are becoming more complex and the sizes of their data sets are rapidly increasing, making the deployment of scalable AI solutions a greater challenge.
```

Comment:
Can we start this differently?  E.g.  "Further developments in artificial intelligence will require better methods for accelerating performance of deep learning workloads"